scripts/grid_search_params.py

```diff
@@ -0,0 +1,65 @@
+import itertools
```

Comment:
Should include a description of what this script does if you're not planning on deleting it prior to production ready status.